afs/afs-mapdb-storage/src/main/java/com/powsybl/afs/mapdb/storage/MapDbAppStorage.java

```diff
@@ -38,13 +39,17 @@ public static MapDbAppStorage createHeap(String fileSystemName) {
     }
 
     public static MapDbAppStorage createMmapFile(String fileSystemName, File dbFile) {
-        DBMaker.Maker maker = DBMaker.fileDB(dbFile);
-        return new MapDbAppStorage(fileSystemName, () -> maker
-                .fileMmapEnable()
-                .fileMmapEnableIfSupported()
-                .fileMmapPreclearDisable()
-                .transactionEnable()
-                .make());
+        return new MapDbAppStorage(fileSystemName, () -> {
+            DBMaker.Maker maker = DBMaker.fileDB(dbFile)
+                    .transactionEnable();
+            // mmap support doesnt' work on Windows
+            if (!SystemUtils.IS_OS_WINDOWS) {
+                maker.fileMmapEnable()
+                        .fileMmapEnableIfSupported()
+                        .fileMmapPreclearDisable();
+            }
```

Comment:
According to the MapDB source code:
- fileMmapEnable change the storeType attribute value to FileMap
- fileMapEnableIfSupported set a boolean flag to true. This flag is used by the make method and change the storeType value to FileMap if the JVM supports large memory map files
- fileMmapPreclearDisable set a flag. This flag is used only if storeType is set to FileMap.

According to that, I think removing the call to fileMmapEnable in the old version of this method should work as expected.